diff --git a/node_modules/expo-modules-core/android/src/main/java/expo/modules/kotlin/KPromiseWrapper.kt b/node_modules/expo-modules-core/android/src/main/java/expo/modules/kotlin/KPromiseWrapper.kt
index a34f9ab..d6c4def 100644
--- a/node_modules/expo-modules-core/android/src/main/java/expo/modules/kotlin/KPromiseWrapper.kt
+++ b/node_modules/expo-modules-core/android/src/main/java/expo/modules/kotlin/KPromiseWrapper.kt
@@ -5,6 +5,9 @@ import expo.modules.kotlin.types.JSTypeConverterProvider
 class KPromiseWrapper(
   private val bridgePromise: com.facebook.react.bridge.Promise
 ) : Promise {
+  private companion object {
+    const val unknownCode = "UnknownCode"
+  }
 
   override fun resolve(value: Any?) {
     bridgePromise.resolve(
@@ -13,6 +16,6 @@ class KPromiseWrapper(
   }
 
   override fun reject(code: String?, message: String?, cause: Throwable?) {
-    bridgePromise.reject(code, message, cause)
+    bridgePromise.reject(code ?: unknownCode, message, cause)
   }
 }
diff --git a/node_modules/expo-modules-core/android/src/main/java/expo/modules/kotlin/Promise.kt b/node_modules/expo-modules-core/android/src/main/java/expo/modules/kotlin/Promise.kt
index 287735e..e249de4 100644
--- a/node_modules/expo-modules-core/android/src/main/java/expo/modules/kotlin/Promise.kt
+++ b/node_modules/expo-modules-core/android/src/main/java/expo/modules/kotlin/Promise.kt
@@ -45,15 +45,15 @@ fun Promise.toBridgePromise(): com.facebook.react.bridge.Promise {
       resolveMethod(value)
     }
 
-    override fun reject(code: String?, message: String?) {
+    override fun reject(code: String, message: String?) {
       expoPromise.reject(code, message, null)
     }
 
-    override fun reject(code: String?, throwable: Throwable?) {
+    override fun reject(code: String, throwable: Throwable?) {
       expoPromise.reject(code, null, throwable)
     }
 
-    override fun reject(code: String?, message: String?, throwable: Throwable?) {
+    override fun reject(code: String, message: String?, throwable: Throwable?) {
       expoPromise.reject(code, message, throwable)
     }
 
@@ -65,15 +65,15 @@ fun Promise.toBridgePromise(): com.facebook.react.bridge.Promise {
       expoPromise.reject(unknownCode, null, throwable)
     }
 
-    override fun reject(code: String?, userInfo: WritableMap) {
+    override fun reject(code: String, userInfo: WritableMap) {
       expoPromise.reject(code, null, null)
     }
 
-    override fun reject(code: String?, throwable: Throwable?, userInfo: WritableMap) {
+    override fun reject(code: String, throwable: Throwable?, userInfo: WritableMap) {
       expoPromise.reject(code, null, throwable)
     }
 
-    override fun reject(code: String?, message: String?, userInfo: WritableMap) {
+    override fun reject(code: String, message: String?, userInfo: WritableMap) {
       expoPromise.reject(code, message, null)
     }
 
