diff --git a/node_modules/expo/src/winter/DOMException.ts b/node_modules/expo/src/winter/DOMException.ts
index 9d8f449..e54cc2a 100644
--- a/node_modules/expo/src/winter/DOMException.ts
+++ b/node_modules/expo/src/winter/DOMException.ts
@@ -63,27 +63,20 @@ const ERROR_CODES: Record<string, number> = {
 };
 
 export class DOMException extends Error {
-  readonly #name: string;
-  readonly #code: number;
-
   constructor(message?: string, name?: string) {
     super(message);
 
-    if (typeof name === 'undefined') {
-      this.#name = 'Error';
-      this.#code = 0;
-    } else {
-      this.#name = String(name);
-      this.#code = ERROR_NAME_TO_ERROR_CODE_MAP[this.name] ?? 0;
-    }
+    const resolvedName = typeof name === 'undefined' ? 'Error' : String(name);
+    this._name = resolvedName;
+    this._code = ERROR_NAME_TO_ERROR_CODE_MAP[resolvedName] ?? 0;
   }
 
   override get name(): string {
-    return this.#name;
+    return this._name;
   }
 
   get code(): number {
-    return this.#code;
+    return this._code;
   }
 }
 
